### DISCUSSION TITLE
How to disable bookmarks in pdf?

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#2655] [Q&A]
In the following minimal example, when I render it in Visual Studio Code in Windows, it generates a PDF with bookmarks (which seems like a bug, as I'm trying to disable the bookmarks, using the `hyperrefoptions:` in YAML)

Here's the screenshot of the bookmark showing up in Sumatra PDF:
> 
![image](https://user-images.githubusercontent.com/7606540/192876520-ebb02b89-a8a5-41bb-816d-e00234b8561c.png)

How can I disable the bookmarks in PDF generated by Quarto?

```markdown
---
format: 
    pdf: 
        hyperrefoptions:
            - bookmarks=false

---

# Title

Here's some text
```

**ETA** a hacky solution found at https://tex.stackexchange.com/questions/13502/how-to-temporarily-disable-pdf-bookmarking is to use `bookmarksdepth=-2` as an option.

**Comment by cderv** [comment]
I believe this will be a sum of several interaction here. 

Quarto uses KOMA script by default: https://quarto.org/docs/output-formats/pdf-basics.html#document-class and the `scrartcl` document class, which recommand using [**bookmark** package ](https://ctan.org/pkg/bookmark). I am not sure this package support the hyperrefoptions `bookmarks=false`

So you could switch to
```
documentclass: article
```

But then, Pandoc will default to use **bookmark** CTAN package if available 
https://github.com/quarto-dev/quarto-cli/blob/d7c07ef4db2ab2fb35b40724802ece8330f76945/src/resources/formats/pdf/pandoc/latex.template#L374

This mean you would need to make sure that **bookmark** package is not available. 

If you do so then 

````
format: 
  pdf:
    documentclass: article
    hyperrefoptions: "bookmarks=false"
````

This above will effectively deactivate bookmarks. 

Maybe this needs a Pandoc feature request to make this line different 
````latex
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}} 
````
and allow to use hyperref only with no bookmark if desired. 

Maybe there is an option to deactivate bookmarks in the CTAN **bookmark** package, but I did not find it , and this is the purpose of this package built above hyperref which state
> One important change of the new driver is that it removed the old hyperref code for book marks and uses the bookmark package instead. So to learn about options to extend the bookmarks you should consult the bookmark documentation too
<sup>([source](https://mirrors.ircam.fr/pub/CTAN/macros/latex/contrib/hyperref/doc/hyperref-doc.html#x1-240006.1.1))</sup>

**Reply by fuhrmanator** [reply]
Thanks for the details as usual!

I tried my MWE with 

```yaml
format: 
  pdf:
    documentclass: article
    hyperrefoptions: "bookmarks=false"
```

and the bookmarks are still there. At least the problem is documented here.
